app/routes/endemics/date-of-visit.js

```diff
@@ -22,11 +22,14 @@ module.exports = [
     path: pageUrl,
     options: {
       handler: async (request, h) => {
-        const { dateOfVisit, landingPage, latestEndemicsApplication } = session.getEndemicsClaim(request)
+        const { dateOfVisit, landingPage, latestEndemicsApplication, typeOfReview } = session.getEndemicsClaim(request)
+        const type = latestEndemicsApplication?.type
         const backLink = landingPage
+        const review = ['R', type].includes(typeOfReview)
```

Comment:
This doesn't quite make sense. type will be 'EE' so will never match typeOfReview which will be either 'R' or 'E'

Reply:
what is the required logic if I may ask

Reply:
I think we have a function called getReviewType that you can use